style.css

```diff
@@ -1,3 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa&display=swap');
+
+* {
+  font-family: 'Comfortaa', cursive;
+}
+
 body {
+  display: grid;
+  justify-content: center;
+  justify-items: center;
+  background: #f4f5f5 url("halloween.png") repeat scroll -2px -5px;
+  background-size: 10rem 10rem;
+}
+
+h1 {
+  border: 1px solid black;
+  background-color: white;
+  padding: 10px 20px;
+}
+
+#books {
+  border: 1px solid black;
+  background-color: white;
+}
```

Comment:
- [ ] This is not optional. At the minimum, every design feature we see on the wireframe should be implemented. Our display of books is lacking the alternating background color on every book container. You are not restricted on color choices though. You can easily achieve this if you used a list to display your books. Nonetheless, you can still do it with `<div>`s. Look at the screenshots below to compare and contrast.

| THE WIREFRAME | YOUR IMPLEMENTATION |
| --- | --- |
| ![WIREFRAME](https://user-images.githubusercontent.com/45002654/138067755-015071bc-149e-4a7c-892e-673856b2a885.png) | ![IMP](https://user-images.githubusercontent.com/45002654/138067795-1b154abf-fc34-410e-b499-820b58b5fc93.png) |